ng2-components/ng2-activiti-tasklist/src/components/task-header.component.ts

```diff
@@ -56,24 +57,48 @@ export class TaskHeaderComponent implements OnChanges {
     refreshData() {
         if (this.taskDetails) {
             const parentInfoMap = this.getParentInfo();
+            let assigneeLabel: any;
+            assigneeLabel = this.translateService.get('TASK_HEADER.ASSIGNEE');
+            let statusLabel: any;
+            statusLabel = this.translateService.get('TASK_HEADER.STATUS');
+            let priorityLabel: any;
+            priorityLabel = this.translateService.get('TASK_HEADER.PRIORITY');
+            let dueDateLabel: any;
+            dueDateLabel = this.translateService.get('TASK_HEADER.DUE_DATE');
+            let categoryLabel: any;
+            categoryLabel = this.translateService.get('TASK_HEADER.CATEGORY');
+            let parentNameLabel: any;
+            parentNameLabel = this.translateService.get('TASK_HEADER.PARENT_NAME');
+            let createdByLabel: any;
+            createdByLabel = this.translateService.get('TASK_HEADER.CREATED_BY');
+            let createdLabel: any;
+            createdLabel = this.translateService.get('TASK_HEADER.CREATED');
+            let idLabel: any;
+            idLabel = this.translateService.get('TASK_HEADER.ID');
+            let descriptionLabel: any;
+            descriptionLabel = this.translateService.get('TASK_HEADER.DESCRIPTION');
+            let formNameLabel:any;
+            formNameLabel = this.translateService.get('TASK_HEADER.FORM_NAME')
+
             this.properties = [
-                new CardViewTextItemModel({ label: 'Assignee', value: this.taskDetails.getFullName(), key: 'assignee', default: 'No assignee', clickable: !this.isCompleted() } ),
```

Comment:
Use the key as label 
label: 'TASK_HEADER.ASSIGNEE' 
and translate the key on the html side with the translate PIPE
i.e. 
card-view-textitem
<div class="adf-property-label">{{ property.label | translate }}</div>